package.json

```diff
@@ -45,6 +45,7 @@
 		"package-plugin:zip-only": "./bin/build-plugin-zip.sh -z",
 		"package-plugin:deploy": "npm run build:deploy && npm run package-plugin:zip-only",
 		"phpunit": "docker-compose up -d db && docker-compose up -d --build wordpress-unit-tests && docker exec -it --workdir /var/www/html/wp-content/plugins/woocommerce-gutenberg-products-block wordpress_test php ./vendor/bin/phpunit",
+		"phpunit:quick": "docker exec -it --workdir /var/www/html/wp-content/plugins/woocommerce-gutenberg-products-block wordpress_test php ./vendor/bin/phpunit",
```

Comment:
I added this because it seems wasteful / slow to `docker up --build` every time we run tests. 

This should be moved to a separate PR, and maybe rationalise the PHPunit commands with other test commands, e.g.

- `npm run test:php:up`
- `npm run test:php`